backends/backends.go

```diff
@@ -64,3 +67,46 @@ func GetFactory(typ string) Factory {
 
 	return factories[normalizeType(typ)]
 }
+
+var globalRequestFieldsByRequestType = map[reflect.Type]map[string]bool{
+	reflect.TypeOf(pb.ReadRequest{}): {
+		"Session":      true,
+		"Backend":      true,
+		"Schema":       true,
+		"DataFormat":   true,
+		"RowLayout":    true,
+		"MultiIndex":   true,
```

Comment:
MultiIndex, Query, GroupBy are only for TSDB
Join, Marker - are not really supported yet

Reply:
Leaving the concept of unsupported operations out for now, as they will fail in this PR as well. The only consideration is whether we want to make a list of globally unsupported operations to make the error message a bit more helpful.